Tensile/Configs/rocblas_sgemm_asm_full.yaml

```diff
@@ -425,6 +425,59 @@ BenchmarkProblems:
           - Range: [ [4], [64, 64, 64, 7000], [1], [3104] ] # skinny-0
           - Range: [ [64, 64, 64, 7000], [4], [1], [3104] ] # skinny-1
 
+  ########################################
+  # NN - VGPR refactor
+  ########################################
+    - # Benchmark Group
+      InitialSolutionParameters:
+      BenchmarkCommonParameters:
+        - EdgeType: ["ShiftPtr"]
+        - LoopTail: [True]
+        - KernelLanguage: ["Assembly"]
+        - PrefetchGlobalRead: [ True ]
```

Comment:
We should explore PGR=0,1 and PLR=0,1.  The "0" option save registers which can be important for these kernels.